pubspec.yaml

```diff
@@ -9,23 +9,15 @@ dependencies:
   flutter:
     sdk: flutter
   equatable: ^2.0.3
+  dartlin: ^0.6.1
   flame:
-    git:
-      url: https://github.com/flame-engine/flame.git
-      path: packages/flame
-      ref: main
+    path: ../flame/packages/flame
```

Comment:
Why path dependencies? Better depend on the branch you created if that is the functionality needed.

Reply:
doens't work with the githubs. the github version of fire atlas depends on path flame. could not get pub get to run, that is why I changed

Reply:
but the build is still busted either way

Reply:
if you know  a way to set it up, please hiajck the pr @spydon. I spent A LOT of time yesterday trying to make it work how it was. also the build was already broken before, the project didn't even compile, so I thought it was ok to leave it broken for now :P

Reply:
Just add this to the bottom of the pubspec file and all path dependencies will use the override instead. :)
```yaml
dependency_overrides:
  flame:
    git:
      url: git@github.com:flame-engine/flame.git
      path: packages/flame
      ref: main
```

Reply:
it had that, but did not work for me!

Reply:
I will untouch this file for merging the PR but would need some help setting it up locally later

Reply:
```dart
[luan@aurora heeve]$ fpg
Error on line 15, column 11: Invalid description in the "flame_fire_atlas" pubspec on the "flame" dependency: "../flame" is a relative path, but this isn't a local pubspec.
   ╷
15 │     path: ../flame
   │           ^^^^^^^^
   ╵
Running "flutter pub get" in heeve...                                   
pub get failed (65;    ╵)
```

I tried with flutter master, stable, everything